rust/kcl-lib/src/parsing/ast/types/mod.rs

```diff
@@ -3230,7 +3230,7 @@ pub enum PrimitiveType {
     /// `fn`, type of functions.
     Function(FunctionType),
     /// An identifier used as a type (not really a primitive type, but whatever).
-    Named(Node<Identifier>),
+    Named { name: Node<Identifier> },
```

Comment:
Bit of a nit, but I think `id` would be better than `name` since the field is an identifier not a name